tests/unit/slack_annotations/core_test.py

```diff
@@ -1,3 +1,177 @@
-class TestHelloWorld:
-    def test_it(self):
-        pass
+import json
+from datetime import UTC, datetime, timedelta
+
+import httpx
+import pytest
+from freezegun import freeze_time
+
+from slack_annotations.core import SEARCH_HOURS, _get_search_after, notify
+
+
+class TestGetSearchAfter:
+    @freeze_time("2024-12-01T01:00:00+00:00")
+    def test_without_cache(self):
+        assert _get_search_after() == "2024-12-01T00:00:00+00:00"
```

Comment:
Could maybe add a comment just to explain what this is testing:

```suggestion
    @freeze_time("2024-12-01T01:00:00+00:00")
    def test_without_cache(self):
        # If there's no cached search_after time it returns one hour before now.
        assert _get_search_after() == "2024-12-01T00:00:00+00:00"
```